modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export function getProperProductsNames(name: string) {
+	if (/dxp/i.test(name)) {
+		return name.replace(/dxp/i, 'Liferay Self-Hosted');
+	}
+
+	if (/lxc - sm/i.test(name)) {
```

Comment:
sort these if statements in alphabetical order

Reply:
Done.